x/compute/client/cli/query.go

```diff
@@ -369,20 +366,24 @@ func GetCmdQuery(cdc *codec.Codec) *cobra.Command {
 			nonce := queryData[:32]
 
 			res, _, err := cliCtx.QueryWithData(route, queryData)
+
 			if err != nil {
-				if strings.Contains(err.Error(), "wasm contract failed: generic: ") {
+				// if this is a query error it contains a
+				if types.ErrContainsQueryError(err) {
 					errorPlainBz, err := wasmCtx.DecryptError(err.Error(), "query", nonce)
 					if err != nil {
 						return err
 					}
-					return fmt.Errorf("%v", errorPlainBz.Error())
-				} else if strings.Contains(err.Error(), "EnclaveErr") {
-					return err
+					return fmt.Errorf("query result: %v", errorPlainBz.Error())
 				}
+				// Itzik: Commenting this as it might have been a placeholder for encrypting
+				//else if strings.Contains(err.Error(), "EnclaveErr") {
+				//	return err
+				//}
 				return err
 			}
 
-			resDecrypted := []byte{}
+			var resDecrypted []byte
```

Comment:
Same same

Reply:
Yep. It was just to silence an annoying IDE warning